### PR TITLE
Import using defaults

### LINKED ISSUE #94: Import does not honour defaults, and causes an import+plan to make changes
When importing Datadog monitors, certain fields (namely `include_tags`) may be missing. These get written not using the default in the schema, but the Go zero value, which may be different.

For example, say the Datadog monitor API returns this JSON (viz, `include_tags` is missing):

```json
{
...
"options": {}
}
```

This gets written as `include_tags = false` even though the default is `true`.

```bash
$ terraform state show datadog_monitor.12345
id                  = 12345
...
include_tags        = false
...
```

If I then try to `terraform plan` with 
```
"resource" "datadog_monitor" "12345" {
...
  "include_tags" = true
}
```

I will see a change:
```bash
$ terraform plan -target datadog_monitor.12345
Refreshing Terraform state in-memory prior to plan...
The refreshed state will be used to calculate this plan, but will not be
persisted to local or remote state storage.

datadog_monitor.12345: Refreshing state... (ID: 12345)

------------------------------------------------------------------------

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  ~ datadog_monitor.12345
      include_tags: "false" => "true"
```

If applied, this writes the new explicit value to Datadog.

I wrote a test that shows this happening (it's rough): https://github.com/terraform-providers/terraform-provider-datadog/commit/496908e41d9d5b1b9bd993c5de21bb9dbfec6eb5

And a (rough) fix: https://github.com/terraform-providers/terraform-provider-datadog/commit/357222fd87636174ae1a12a8e45e64f9a17e3d74

### Terraform Version

```
Terraform v0.11.8
+ provider.datadog v1.2.0
```

### Affected Resource(s)

- datadog_monitor

### PR DESCRIPTION
**Pull request by nickatsegment**
Fixes #94 

This is my first terraform PR. Extra scrutiny appreciated. :)

### CODE REVIEW

File: datadog/resource_datadog_monitor.go [hunk 1]
```diff
@@ -331,7 +332,15 @@ func resourceDatadogMonitorRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("timeout_h", m.Options.GetTimeoutH())
 	d.Set("escalation_message", m.Options.GetEscalationMessage())
 	d.Set("silenced", m.Options.Silenced)
-	d.Set("include_tags", m.Options.GetIncludeTags())
+	if includeTags, ok := m.Options.GetIncludeTagsOk(); ok {
```
Comment (pdecat) [inline datadog/resource_datadog_monitor.go]: I believe the real issue is here: https://github.com/terraform-providers/terraform-provider-datadog/blob/74b24b9c212b3691e7fa055d37aea96cdd17edb5/vendor/github.com/zorkian/go-datadog-api/datadog-accessors.go#L10560

File: datadog/resource_datadog_monitor.go [hunk 2]
```diff
@@ -331,7 +332,15 @@ func resourceDatadogMonitorRead(d *schema.ResourceData, meta interface{}) error
...
+		d.Set("include_tags", includeTags)
+	} else {
+		default_, err := res.Schema["include_tags"].DefaultValue()
```
Comment (pdecat) [inline datadog/resource_datadog_monitor.go]: This `else` clause is not needed, terraform will do the job of populating the default.

File: datadog/resource_datadog_monitor.go (hunk 1 above)
Comment (pdecat) [reply datadog/resource_datadog_monitor.go]: Using `GetIncludeTagsOk()` seems the right thing to do right now.

**Comment by nickatsegment** [comment]
I'm not especially involved in this area of work any more, so I can't commit to pushing this over the line and making sure it works.

**Pull request closed without merging**